app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -8,7 +8,7 @@
       p
         'A one-time passcode has been sent to <strong>#{@phone_number}</strong>.
         'Please enter the code that you received.
-        'If you do not receive the code in #{otp_drift_time_in_minutes} minutes, please #{link_to 'request a new passcode', users_otp_new_path, data: { 'no-turbolink' => true }}.
+        'If you do not receive the code in #{otp_valid_for_in_words}, please #{link_to 'request a new passcode', users_otp_new_path, data: { 'no-turbolink' => true }}.
 
```

Comment:
I think that might be more than they user really needs to know here.  Perhaps a better approuch would be that if the user has an expired OTP, and  they try to enter one, it could say "You current one time code has expired since it was not used in X minutes.  A new one was sent to your phone"?